### PR TITLE
update grb filter, add new test data

### LINKED ISSUE #149: [MM module] Update / New filters
* Update the on_axis_grb filters
remove the alerts with a realbogus score < 0.9 for the gold filter
remove the gcn with an error area above 5 degree for the gold filter

* Add GW filters

### PR DESCRIPTION
**Pull request by FusRoman**
**IMPORTANT: Please create an issue first before opening a Pull Request.**
Linked to issue(s): close #149 

## What changes were proposed in this pull request?

How is the issue this PR is referenced against solved with this PR?

### CODE REVIEW

**Comment by JulienPeloton** [comment]
Ready for the review?

**Comment by FusRoman** [comment]
yes

File: fink_filters/filter_mm_module/filter.py [hunk 1]
```diff
@@ -42,37 +47,39 @@ def bronze_events(fink_class, realbogus_score):
...
     >>> len(df[df["f_bronze"]])
-    25
+    4
     """
-    f_bogus = realbogus_score >= 0.5
+    f_obs = observatory.isin(GRB_OBSERVATORY)  # select only the GRB observatories
+
+    f_bogus = rb >= 0.7
 
     f_class = fink_class.isin(["SN candidate", "Unknown", "Ambiguous"])
```
Comment (JulienPeloton) [inline fink_filters/filter_mm_module/filter.py:58]: Here is my suggestion:
```python
from fink_utils.xmatch.simbad import return_list_of_eg_host
...
base_extragalactic = return_list_of_eg_host() # include Unknown and Fail as well
fink_extragalactic = ["SN candidate", "Early SN Ia candidate", "Ambiguous"]
extragalactic = base_extragalactic + fink_extragalactic
f_class = fink_class.isin(extragalactic)
```

File: fink_filters/filter_mm_module/filter.py [hunk 2]
```diff
@@ -153,38 +162,98 @@ def gold_events(fink_class, realbogus_score, grb_proba, rate):
...
+    >>> df = pd.read_parquet(grb_output_data)
+    >>> df["f_bronze"] = gw_bronze_events(df["fink_class"], df["observatory"], df["rb"])
+    >>> len(df[df["f_bronze"]])
+    0
+    """
+    f_obs = observatory.isin(GW_OBSERVATORY)  # select only the GRB observatories
+
+    f_bogus = rb >= 0.7
+
+    f_class = fink_class.isin(["SN candidate", "Unknown", "Ambiguous"])
```
Comment (JulienPeloton) [inline fink_filters/filter_mm_module/filter.py:227]: cf extragalactic definition above

**Review by JulienPeloton — APPROVED** [review]
OK for me -- but I suggest an extension of allowed classes.

File: fink_filters/filter_mm_module/filter.py (hunk 1 above)
Comment (JulienPeloton) [reply fink_filters/filter_mm_module/filter.py:58]: ah I forgot the KN class:

```
fink_extragalactic = ["KN candidate", "SN candidate", "Early SN Ia candidate", "Ambiguous"]
```

**Comment by FusRoman** [comment]
I think, we are ready to merge.
last review @JulienPeloton ?

**Comment by JulienPeloton** [comment]
LGTM -- you can merge ! Thanks.

**Pull request merged**